.github/workflows/scripts/auto-update-version.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash -e
+
+repo=$1
+variable_name=$2
+file_names=("${@:3}") # remaining args
+
+latest_version=$(gh api -q .tag_name "repos/open-telemetry/$repo/releases/latest" | sed 's/^v//')
+
+echo "Repo: $repo"
+echo "Latest version: $latest_version"
+
+for file_name in "${file_names[@]}"
+do
+  sed -i -e "s/$variable_name: .*/$variable_name: $latest_version/" "$file_name"
+done
+
+if git diff --quiet; then
+  echo "Already at the latest version."
+  exit 0
+fi
+
+message="Update $repo version to $latest_version"
+body="Update $repo version to \`$latest_version\`."
```

Comment:
It would be helpful if the body could contain, at a minimum, a link to the repo's releases; or even better, a list to the specific version. Or both :)

Reply:
done

Reply:
Is this a change you'll be adding in this or a followup PR @trask (or maybe have someone else do it)?

Reply:
Oh, sorry, I missed that. Thanks!